lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -50,9 +50,8 @@ TurboGraft.handlers.remoteFormHandler = (ev) ->
 documentListenerForButtons = (eventType, handler, useCapture = false) ->
   document.addEventListener eventType, (ev) ->
     target = ev.target
-    return if !target
 
-    while target != document && (typeof target != "undefined")
+    while target != document && (typeof target != "undefined") && (target != null)
```

Comment:
My rationale was that `target != null` will cause an error is target is not defined, whereas (I think anyways) that `typeof target != "undefined"` will return false even if target is not defined. Therefore, if target was undefined, or null, the checked would be false, an not in an error state. That being said, in this context I don't see how target would be undefined, so I agree with your solution. Updated in latest commit, thanks!